capitolwords_ng/parsers/text_utils.py

```diff
@@ -26,10 +34,13 @@ def preprocess(text):
     text = re.sub('HON\.', 'HON', text)
```

Comment:
Can you include an inline comment for each of these regexes to explain what they're for?